server/public/model/session.go

```diff
@@ -39,6 +39,13 @@ const (
 //msgp:tuple StringMap
 type StringMap map[string]string
 
+type MobileVersionMetric struct {
+	Version              string
+	Platform             string
+	Count                float64
+	NotificationDisabled string
+}
```

Comment:
Do we want the `NotificationDisabled` associated to the version though? It seems this would duplicate cardinality. We should be good to just have a total count of disabled ones regardless of version

Reply:
We have several options here.

If we look into prometheus side, we can have two different metrics:
- Mobile versions / platform
- Notifications disabled / platform

If we do that instead of `X * 3 * 2` we have `(X * 3) + (2 * 3)` which should be better in terms of cardinality. Now... there are two ways to achieve that.
- Get the data as we are now, and compose it in Golang
- Make two database requests, one for each metric

In this case I have the gut feeling that it is better to get everything, and just compose the values in Golang.

All that being said... we only store one set of values every day, so not sure how "dangerous" is the cardinality in this situation. So right now I lean towards keeping the code simple. But if you have a strong opinion about reducing the cardinality, I can go with 2 metrics and composition in Golang.

Reply:
You are right, I was thinking more of the problem of cardinality with labels but forget we are not storing this as labels of other values so I'm good with your original approach 👍